getting-started/kubernetes/apiserver-preview.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Try out the Calico API server tech-preview
+description: Install the Calico API server on an existing Calico cluster
+canonical_url: '/getting-started/kubernetes/quickstart'
+---
+
+### Big picture
+
+[ **Feature status**: Tech-preview in v3.19 ]
+
+Install the Calico API server on an existing cluster to enable management of Calico APIs using kubectl.
+
+> **Warning!** The Calico API server is a tech preview feature and should not be used in production clusters. It has had limited testing and is liable to change in future releases. Please provide feedback and issue reports via GitHub.
+{: .alert .alert-danger }
+
+### Value
+
+Normally, calicoctl is required in order to manage Calico APIs in the `projectcalico.org/v3` API group. The API server provides a REST API for Calico, and allows management of these APIs using kubectl instead.
+
+### Before you begin
+
+- A cluster with Calico installed using the Kubernetes API data store.
+
+- A machine with `openssl` installed.
+
+### Concepts
+
+#### calicoctl vs kubectl
+
+In previous releases, calicoctl has been required to managed Calico API resources in the `projectcalico.org/v3` API group. The calicoctl CLI tool provides important validation and defaulting on these APIs. The Calico API server performs
+that defaulting and validation server-side, exposing the same API semantics without a dependency on calicoctl.
+
+calicoctl is stil required for the following subcommands:
+
+- [calicoctl node]({{site.baseurl}}/reference/calicoctl/node)
+- [calicoctl ipam]({{site.baseurl}}/reference/calicoctl/ipam)
+- [calicoctl convert]({{site.baseurl}}/reference/calicoctl/convert)
+- [calicoctl version]({{site.baseurl}}/reference/calicoctl/version)
+
+### How to
+
+#### Install the API server
+
+1. Create the following manifest, which will install the API server as a deployment in the `tigera-system` namespace.
```

Comment:
Is the "tigera" here intended?  I think it would be the first time we're using the Tigera name in Calico OSS - which if true feels like a big branding step.  (There are also lots more "tigera"s in the code fragments below.)

Reply:
Technically we have already done that with "tigera-operator" but this is a bit different. 

It's only in `tigera-system` right now because it was the easiest thing to get working, but I am in conversations with @tmjd about how / where we want to place this long term.

Reply:
As long as it's being considered, I'm sure you'll work out the right thing!